infra/charts/feast-feature-server/samples/service-monitor.yml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.metrics }}
```

Comment:
given that these are samples, no longer instrumented by the chart, these if statements should be removed.